resources/quality/z-bolt/direct/ABS/v6 0.3/z-bolt_direct_v6_0.30_ABS_extrafine.inst.cfg

```diff
@@ -0,0 +1,23 @@
+[general]
+version = 4
+name = Extra Fine
+definition = z-bolt_base
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = extrafine
+material = generic_abs
+variant = Direct E3DV6 0.30mm
+
+[values]
+speed_print = 25
+retraction_amount = 1.1
+adhesion_type = brim
+
+cool_fan_speed_min = 35
+cool_fan_speed_max = 60
+cool_min_layer_time_fan_speed_max = 20
```

Comment:
It may be worth considering - for your own sanity - to override this setting in the ABS profile instead of in ALL of the quality profiles. For ABS you're using 20, and for all of the rest 15 so the printer definition could just set it to 15 and have ABS override it.

Reply:
How do we add cool_min_layer_time and other settings to material profiles?

Reply:
The ABS material profile is here: https://github.com/Ultimaker/fdm_materials/blob/master/generic_abs_175.xml.fdm_material

The material profile format has a number of specific settings (like `print cooling` instead of Cura's `cool_fan_speed`), but `cool_min_layer_time` is not one of them. Cura's own settings can be set by adding the `cura` namespace to the XML tag, like so:

```
<cura:setting key="cool_min_layer_time">10</cura:setting>
```

Reply:
Thank you a lot for this answer, we had a lot of struggle with those advanced material profile settings.